### PR TITLE
Bugfix/spline 874 admin cli - fix ANSI codes rendering

### PR DESCRIPTION
**Pull request by wajda**

### CODE REVIEW

File: commons/src/main/scala/za/co/absa/spline/common/ConsoleUtils.scala [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      val ansiAllowed = System.console != null && System.getenv.get("TERM") != null
+      val ansiString  = $ansiInterpolation
+
+      if (ansiAllowed) ansiString
+      else stripAnsiEscapeSequences(ansiString)
+    """
+  }
+
+  def stripAnsiEscapeSequences(s: String): String = {
+    val sciSequencePattern = "\u001b\\[.*?[\\x40-\\x7E]"
```
Comment (cerveada) [inline commons/src/main/scala/za/co/absa/spline/common/ConsoleUtils.scala]:
```suggestion
    val ansiSequencePattern = "\u001b\\[.*?[\\x40-\\x7E]"
```
Is this typo?

**Review by cerveada — DISMISSED** [review]

File: commons/src/main/scala/za/co/absa/spline/common/ConsoleUtils.scala (hunk 1 above)
Comment (wajda) [reply commons/src/main/scala/za/co/absa/spline/common/ConsoleUtils.scala]: Ah, yes, it's a typo :)
Should be CSI (Control Sequence Introducer)
https://en.wikipedia.org/wiki/ANSI_escape_code#CSI_(Control_Sequence_Introducer)_sequences

Comment (wajda) [reply commons/src/main/scala/za/co/absa/spline/common/ConsoleUtils.scala]:
```suggestion
    val csiSequencePattern = "\u001b\\[.*?[\\x40-\\x7E]"
```

**Comment by wajda** [comment]
Tested in Linux as well as Windows -- standard terminal, PS and Cygwin (thanks @dk1844)

**Pull request merged**